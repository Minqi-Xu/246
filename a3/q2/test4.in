f 5
r
f 3
r
c
t
m
