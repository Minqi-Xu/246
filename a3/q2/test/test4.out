Current position: (3,5), Facing South
8
8
